main/main.cpp

```diff
@@ -3564,6 +3567,8 @@ int Main::start() {
 			if (parsed_pair) {
 				E = E->next();
 			}
+		} else if (E->get().begins_with("--export-")) {
+			ERR_FAIL_V_MSG(EXIT_FAILURE, "Please provide a valid preset when exporting, aborting.");
```

Comment:
You could add a check for project, for different message when project is not found.

Reply:
Done

Reply:
You just shortened it. It doesn't tell whether the project is missing.

Reply:
what about something like "Please provide a export preset target when exporting"

Reply:
Well on another though you can just tweak it a bit to say "Missing export preset name, aborting.", so it's clear that it's a missing argument, not a wrong preset.